tools/test_checks.sh

```diff
@@ -43,6 +43,12 @@ if uname | grep -q 'Darwin' 2>&1; then
     opt_config_target='--target=darwin'
 fi
 
+# Fix SVE2 issues
```

Comment:
Instead of detecting an older GAS and forcing users to force a build anyway, why not just print a warning and continue?  This is what other arch are doing and won't require to do all again in the test scripts.  Or if it is really required then we wouldn't need workarounds in the test scripts.

Reply:
agree. reworked.